src/doc_builder/autodoc.py

```diff
@@ -317,9 +317,9 @@ def document_object(object_name, package, page_info, full_name=True):
             check = quality_check_docstring(object_doc, object_name=object_name)
             object_doc = convert_md_docstring_to_mdx(obj.__doc__, page_info)
 
-        source_link = get_source_link(obj, page_info)
-        component = get_signature_component(signature_name, anchor_name, signature, object_doc, source_link)
-        documentation += "\n" + component + "\n"
+    source_link = get_source_link(obj, page_info)
+    component = get_signature_component(signature_name, anchor_name, signature, object_doc, source_link)
+    documentation = "\n" + component + "\n"
```

Comment:
because of the if condition above and indentation, nothing was generated if `obj.__doc__` was empty.

With this change, we would see, something like this
<img width="500" alt="Screenshot 2022-02-15 at 14 07 35" src="https://user-images.githubusercontent.com/11827707/154069272-528821d9-eb2f-4fbb-b3ef-bc4c3993a3d2.png">